api/Routes/PresidentRoutes.cs

```diff
@@ -56,43 +58,50 @@ public static void RegisterPresidentApi(WebApplication app)
 
                 return Results.Ok(president);
             })
+            .Produces<President?>(200)
             .WithMetadata(new SwaggerOperationAttribute(
                 summary: PresidentEndpointMetadataMessages.MESSAGE_PRESIDENT_BYNAME_SUMMARY,
                  description: PresidentEndpointMetadataMessages.MESSAGE_PRESIDENT_BYNAME_DESCRIPTION
                  ));
 
             app.MapGet($"{API_PRESIDENT_ROUTE_COMPLETE}/year/{{year}}", async (int year, DBContext db) =>
             {
+                //Require validation to not year greater than actual year
```

Comment:
This validation is not required in that case it should return not found. pls remove the comment

Reply:
I mean in this one, is if is a 1000 year no president must be search, the minimum year must be the minimum date stored, the same for future year calls for future year must be stoped before go to db

Reply:
it doesn't matter. This is a bad use of the API not an scenario that we need to validate. not found is a valid response for those actions.